README.md

```diff
@@ -4,6 +4,8 @@
 
 * This project is fully sponsored by the [Game Preservation Society](https://www.gamepres.org/en/).
 
+[OpenAL](https://www.openal.org/) is required to run this application. Please install it using the most recent instructions for your operating system of choice.
```

Comment:
Not living to the philosophy of Aaru.

Reply:
Then please give a suggestion on how to improve this.

Reply:
As per offline conversation, this may be something that is handled later when we end up with an installer. If that is a reasonable solution, then I believe all of the current feedback is handled.

Reply:
Also, unfortunately, we are relying on a branch version of CSCore for this cross-platform functionality. It's not an ideal situation.